cspell-custom-words.txt

```diff
@@ -0,0 +1,194 @@
+Acala
+acala
+Acks
+Alertmanager
+algod
+Algorand
+algosdk
+alist
+Aptos
+authorisation
+authorise
+authorised
+backdoors
+Bech
+bech
+behaviour
+bigset
+Bigtable
+borsh
+bscscan
+BUILDKIT
+bytecodes
+callstack
+CCTP
+Celestia
+celestia
+certusone
+Chainlink
+Coinspect
+collateralization
+colour
+Concat
+conftest
+Cosm
+cosmoshub
+Cosmwasm
+cosmwasm
+counterparty
+cpus
+crosschain
+Cyfrin
+datagram
+denoms
+devnet
+Dymension
+dymension
+EMMITTER
```

Comment:
`EMMITTER` seems incorrect and should be changed :upside_down_face:

Reply:
Fixed in b9adf2d21bfdf669ba00a3a50c5c1179cd152a85